.github/workflows/autoreview.yaml

```diff
@@ -22,20 +22,20 @@ jobs:
 
     steps:
       - name: Checkout code
-        uses: actions/checkout@v2
```

Comment:
at the very least those changes look safe enough to be extracted to reduce the noise?

Reply:
yeah. Going to revert them